https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/8238

From e76098268a833f697336d718d38ce911a8fd38fb Mon Sep 17 00:00:00 2001
Message-ID: <e76098268a833f697336d718d38ce911a8fd38fb.1736022660.git.sam@gentoo.org>
In-Reply-To: <30bce05f5cc6ed0b0f2cf43fcb5558900eeaa198.1736022660.git.sam@gentoo.org>
References: <30bce05f5cc6ed0b0f2cf43fcb5558900eeaa198.1736022660.git.sam@gentoo.org>
From: Sam James <sam@gentoo.org>
Date: Sat, 4 Jan 2025 20:30:01 +0000
Subject: [PATCH 2/2] codec2json: move option to right section

It has an external dependency (json-glib) so should be under
that heading.

Fixes: fd588a50e415feb0ab21c4a3386bd426c8c9043b
---
 meson_options.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson_options.txt b/meson_options.txt
index b3ec11e911..568871f42d 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -15,7 +15,6 @@ option('audiovisualizers', type : 'feature', value : 'auto')
 option('autoconvert', type : 'feature', value : 'auto')
 option('bayer', type : 'feature', value : 'auto')
 option('camerabin2', type : 'feature', value : 'auto')
-option('codec2json', type : 'feature', value : 'auto')
 option('codecalpha', type : 'feature', value : 'auto')
 option('codectimestamper', type : 'feature', value : 'auto')
 option('coloreffects', type : 'feature', value : 'auto')
@@ -101,6 +100,7 @@ option('bs2b', type : 'feature', value : 'auto', description : 'Bauer stereophon
 option('bz2', type : 'feature', value : 'auto', description : 'bz2 stream encoder and decoder plugin')
 option('chromaprint', type : 'feature', value : 'auto', description : 'Chromaprint fingerprint audio plugin')
 option('closedcaption', type : 'feature', value : 'auto', description : 'Closed caption extractor, decoder, and overlay plugin')
+option('codec2json', type : 'feature', value : 'auto')
 option('colormanagement', type : 'feature', value : 'auto', description : 'Color management correction plugin')
 option('curl', type : 'feature', value : 'auto', description : 'cURL network source and sink plugin')
 option('curl-ssh2', type : 'feature', value : 'auto', description : 'cURL network source and sink plugin libssh2 support')
--
2.47.1
